content/rs/getting-started/quick-setup.md

```diff
@@ -34,6 +34,10 @@ the current directory.
 $ sudo ./install.sh -y
 ```
 
+Note: RS installation on Ubuntu 18.04 fails if port 53 is used by systemd-resolved 
```

Comment:
"In default Ubuntu 18.04 installations, port 53 is in use by systemd-resolved (DNS server). In such a case, the system configuration must be changed to make this port available before running RS installation."

Reply:
I used:
When port 53 is in use, the installation fails. This is known to happen in default Ubuntu 18.04 installations in which systemd-resolved (DNS server) is running. To workaround this issue, change the system configuration to make this port available before running RS installation.

Reply:
@Paz-Yanover Is this above note OK? Wanted to cover any situation where port 53 is in use.